apps/nextjs/src/app/dashboard/layout.tsx

```
@@ -0,0 +1,42 @@
+  children,
+  sidebar,
+}: Readonly<DashboardLayoutProps>) {
+  return (
+    <BreadCrumbsProvider>
+      <SidebarProvider className="border-[1px]">
+        {sidebar}
+        <SidebarInset className="border-l-[1px] bg-transparent">
+          <DashboardHeader />
+          <Suspense fallback={<DashboardContentSkeleton />}>
+            <section
+              role="status"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<output>


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)